privaterelay/utils.py

```diff
@@ -473,3 +482,42 @@ def flag_is_active_in_task(flag_name: str, user: AbstractBaseUser | None) -> boo
             return True
 
     return False
+
+
+class VersionInfo(TypedDict):
+    source: str
+    version: str
+    commit: str
+    build: str
+
+
+@cache
+def get_version_info(version_json_path: Path | None = None) -> VersionInfo:
+    """Return version information written by build process."""
+    version_json_path = version_json_path or (Path(settings.BASE_DIR) / "version.json")
+    info = {}
+    if version_json_path.exists():
+        with version_json_path.open() as version_file:
+            try:
+                info = json.load(version_file)
+            except ValueError:
+                pass
+            if not hasattr(info, "get"):
+                info = {}
+    version_info = VersionInfo(
+        source=info.get("source", "https://github.com/mozilla/fx-private-relay"),
+        version=info.get("version", "unknown"),
+        commit=info.get("commit", "unknown"),
+        build=info.get("build", "not built"),
+    )
+    return version_info
```

Comment:
praise: ooo, nice helper! I want to use this to include the version in the google cloud profiler data too! (https://github.com/mozilla/fx-private-relay/pull/4321/files#diff-26ec47835bb87054210dfba58b8590e7d2bb34d187de1d791d9d8c9c00a7baafR24-R31)